extensions/objects/templates/image-with-regions-editor.component.ts

```diff
@@ -314,6 +314,16 @@ export class ImageWithRegionsEditorComponent implements OnInit {
   }
 
   getPreviewUrl(imageUrl: string): string {
+    if (
+      this.contextService.getEntityType() !== (
+        AppConstants.ENTITY_TYPE.EXPLORATION)
+    ) {
+      const entityType: string = this.contextService.getEntityType() as string;
```

Comment:
You can do this with a single function call to `this.contextService.getEntityType()`. Use a separate variable before the if condition.

Reply:
Done